code/modules/uplink/uplink_items.dm

```diff
@@ -1830,8 +1806,6 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/vehicle/sealed/car/clowncar
```

Comment:
Ah yes, the clown wordlessly swallowing everyone standing in the halls and thenending up doing something stupid that either results in everyone involved dying or being dusted, or the clown being turned into a sausage. This has much more potential for RDM than it has for roleplay.